lib/mocha.js

```diff
@@ -440,18 +407,49 @@ Mocha.prototype.fullTrace = function() {
 };
 
 /**
- * Enables desktop notification support.
+ * Enables desktop notification support if prerequisite software installed.
  *
  * @public
+ * @see {@link Mocha#isGrowlCapable}
  * @see {@link Mocha#_growl}
  * @return {Mocha} this
  * @chainable
  */
 Mocha.prototype.growl = function() {
-  this.options.growl = true;
+  this.options.growl = this.isGrowlCapable();
+  if (!this.options.growl) {
+    var detail = process.browser
+      ? 'Growl notification support not available in browser... cannot enable!'
+      : 'Growl notification support not installed... cannot enable!';
```

Comment:
How about adding [installation guide](https://github.com/tj/node-growl#installation) in this error message?

Reply:
Think that would be better suited for website than source code. Although now that you mention it, the user guide should link to the [wiki](https://github.com/mochajs/mocha/wiki/Growl-Notifications).

Reply:
Users can see this message when they are using `--growl` without growl.
So, it's better that let users know how to install growl notification when they try to use growl notification than website or wiki.

Reply:
They see every other error message Mocha generates as well, but you didn't add links for any of those.

Although it might be interesting to add a page to our website called "So I Got this Error..."

We have a website for this. I don't mind converting the existing wiki page into a user guide addition. TJ's instructions are _outdated_ for macOS (I rewrote them 6mos ago). Since half a year went by before anyone noticed, why would we want to redirect _anyone_ there?